docs/manual/troubleshooting.tex

```diff
@@ -968,6 +968,44 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
 \end{enumerate}
 
 
+\subsection{Configure Eclipse to Edit Checker Framework\label{building-eclipse}}
+\begin{enumerate}
+\item
+Install Eclipse from the Eclipse official website.
+
+\item
+Once installation is complete, run Eclispe and set the working directory to jsr308.
+
+\item Try locating the "plume-lib" file in jsr308. If plume-lib already exists, ignore this step. Otherwise, depending on the organization you are working from(typetools, opprop, or pascaliUWat), git-clone plume-lib by: (change "typetools" in git-clone URL below to desired organization)
```

Comment:
Multiple organizations aren't mentioned anywhere else in this section, so I wouldn't mention it here.